### PR TITLE
Switch to Vite to build, AdvantageScope's NT4 Library, and CopperConsole network table as a data source.

### PR DESCRIPTION
**Pull request by aidnem**
Changes:
- electron-forge is still used, but it is now using the Vite plugin to handle bundling, which gives more flexibility when using multiple files (especially when trying to import multiple files on the renderer process). The build process is much more streamlined now, simply requiring you to run `npm run start` or `npm run make` (to make an executable) without having to run `tsc` beforehand.
- AdvantageScope's NT4 library is now used. It supports NetworkTables V4, instead of the previous network tables v2 library that was used. This should work much better than before.
- The `CopperConsole` network table is now used, instead of stealing data from other apps' network tables. There is a specification defined in `nt-specification.md`.

Closes issues:
- #8 - Add CopperConsole network table specification
- #9 - Migrate to Vite
- #7 - Use network tables version 4

### CODE REVIEW

File: package-lock.json
Comment (jkleiber) [inline package-lock.json:1]: Do we need to check in this lock file?

**Review by jkleiber — APPROVED** [review]
One non-blocking comment. approved

File: package-lock.json
Comment (aidnem) [reply package-lock.json:1]: @jkleiber I believe so, based on a few quick google searches. [This medium article](https://medium.com/@InspireTech/should-i-add-package-lock-json-to-git-why-is-it-important-to-commit-package-lock-json-into-git-e8273a8aad00) contains a few reasons why.

Comment (jkleiber) [reply package-lock.json:1]: Sounds good

**Pull request merged**